### ISSUE TITLE
the --output $filename doesn't actually write to file

### ISSUE THREAD

**Issue by kalip2** [twpayne/chezmoi#3566]
## Describe the bug

When doing an `update`, the output that typically goes to stdout doesn't go to my chezmoi.log file. 

## To reproduce

Run the following: `chezmoi -v update --output chezmoi.log`

## Expected behavior
I should be able to run `cat chezmoi.log` and see all the output that typically goes to `stdout`

## Output of command with the `--verbose` flag

```console
$ chezmoi --verbose $COMMAND
```

## Output of `chezmoi doctor`

```console
❯ chezmoi doctor
RESULT    CHECK                       MESSAGE
ok        version                     v2.46.0, commit 1418f74ab40bbee807a1438ec0bbdb6c85f24a74, built at 2024-01-25T17:18:45Z, built by Homebrew
warning   latest-version              v2.46.1
ok        os-arch                     darwin/arm64
ok        uname                       Darwin macbookairchiefexec.lan 23.3.0 Darwin Kernel Version 23.3.0: Wed Dec 20 21:33:31 PST 2023; root:xnu-10002.81.5~7/RELEASE_ARM64_T8112 arm64
ok        go-version                  go1.21.6 (gc)
ok        executable                  /opt/homebrew/bin/chezmoi
ok        upgrade-method              brew-upgrade
ok        config-file                 no config file found
ok        source-dir                  ~/.local/share/chezmoi is a git working tree (clean)
ok        suspicious-entries          no suspicious entries
ok        working-tree                ~/.local/share/chezmoi is a git working tree (clean)
ok        dest-dir                    ~ is a directory
ok        umask                       022
ok        cd-command                  found /opt/homebrew/bin/fish
ok        cd-args                     /opt/homebrew/bin/fish
info      diff-command                not set
ok        edit-command                found /opt/homebrew/bin/code
ok        edit-args                   code --wait --new-window
ok        git-command                 found /usr/bin/git, version 2.39.3
ok        merge-command               found /usr/bin/vimdiff
ok        shell-command               found /opt/homebrew/bin/fish
ok        shell-args                  /opt/homebrew/bin/fish
info      age-command                 age not found in $PATH
info      gpg-command                 gpg not found in $PATH
info      pinentry-command            not set
ok        1password-command           found /opt/homebrew/bin/op, version 2.24.0
info      bitwarden-command           bw not found in $PATH
info      bitwarden-secrets-command   bws not found in $PATH
info      dashlane-command            dcli not found in $PATH
info      doppler-command             doppler not found in $PATH
info      gopass-command              gopass not found in $PATH
info      keepassxc-command           keepassxc-cli not found in $PATH
info      keepassxc-db                not set
info      keeper-command              keeper not found in $PATH
info      lastpass-command            lpass not found in $PATH
info      pass-command                pass not found in $PATH
info      passhole-command            ph not found in $PATH
info      rbw-command                 rbw not found in $PATH
info      vault-command               vault not found in $PATH
info      vlt-command                 vlt not found in $PATH
info      secret-command              not set
```

</details>

**Comment by twpayne** [comment]
`chezmoi update` calls `chezmoi apply`, and `chezmoi apply` can be interactive. For example, it might prompt for a password for a password manager, prompt you that a file has been modified since chezmoi last wrote it, or invoke a custom pager.

Therefore, `--output` does not apply to `chezmoi update`. If you're sure that `chezmoi update` will not require interaction and you want to overwrite all changes, then you can use:

```console
$ chezmoi update -v --force >chezmoi.log
```

**Comment by twpayne** [comment]
Thank you for reporting this. This is a bug in chezmoi's documentation which is fixed in 029de9ad4952d9394bd0dd6384fe794ff2ed84da.
